### ISSUE TITLE
Extra content

### ISSUE THREAD

**Issue by Belomi** [bryanforbes/Erasmus#58]
When a verse or chain of verses is cited the phrase "Read full chapter" is appended. This may be the result of a update to BibleHub.

**Comment by bryanforbes** [comment]
Can you give me a usage example where this appears?

**Comment by StargazeSparkle** [comment]
`$ John 1:1`
yields
```
1. In the beginning was the Word, and the Word was with God, and the Word was God. Read full chapter
```

**Comment by bryanforbes** [comment]
Thank you! I'll try to get a fix for this tonight.

**Comment by bryanforbes** [comment]
This was fixed via 1c9ab161ead1887be2a30960b036a16072855c65
